inbm-lib/inbm_common_lib/shell_runner.py

```diff
@@ -105,15 +107,26 @@ def run_with_log_path(self,
                 shell=False,
                 stdin=subprocess.DEVNULL,
                 stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT)
+                stderr=subprocess.STDOUT,
+                env=env)
+        elif env:
+            proc = subprocess.Popen(
+                cmd,
+                cwd=cwd,
+                shell=True,
```

Comment:
we cannot run with shell=True; that's the purpose of this class, to mimic a shell without its security issues

Reply:
Changes reverted. No shell=True now.